docs/adrs/0549-composite-run-steps.md

```diff
@@ -250,9 +250,21 @@ If any of the steps fail in the composite action and the `continue-on-error` is
 
 For the composite action steps, it follows the same logic as above. In this example, `"Hello World 2"` will be outputted because the previous step has `continue-on-error` set to `true` although that previous step errored. 
 
-### Defaults
+### Defaults, Shell, and Working-dir
+We will not support "defaults" in a composite action. 
```

Comment:
may need to escape `"` as `\"`

Reply:
Just double checked, since it's a .md, I believe you don't need to escape. I double checked in the preview and it looks as intended